src/components/structures/MatrixChat.js

```diff
@@ -139,6 +139,10 @@ module.exports = React.createClass({
         return this.props.config.default_is_url || "https://vector.im";
     },
 
+    getEnableLabs() {
```

Comment:
bit pointless? just pass through `this.props.config.enableLabs` and let UserSettings deal with the rest?